Library/Homebrew/extend/ENV/super.rb

```diff
@@ -164,7 +164,11 @@ def determine_isystem_paths
   end
 
   def determine_include_paths
-    keg_only_deps.map { |d| d.opt_include.to_s }.to_path_s
+    paths = keg_only_deps.map { |d| d.opt_include.to_s }
+    if compiler == :llvm_clang && Formula["llvm"].installed?
```

Comment:
Why check if it is installed? If it's set as the compiler it's going to be used regardless, no? Also, should this be dependent on OpenMP being used? No worries if not, genuinely asking.